hq/app/logic/SnykDisplay.scala

```diff
@@ -0,0 +1,52 @@
+package logic
+
+import play.api.libs.json._
+import play.api.libs.json.Reads._
+import utils.attempt.{Attempt, FailedAttempt, Failure}
+
+import scala.concurrent.ExecutionContext
+
+import model._
+
+object SnykDisplay extends Serializers {
+
+  def getOrganisationId(s: String, organisation: Organisation) = {
+    val id = for {
+      orglist <- (Json.parse(s) \ "orgs").asOpt[List[SnykOrg]]
```

Comment:
It looks like there are some unhandled error cases in this, e.g. parsing failure. We want to try and avoid exceptions getting thrown so this might just need a try block.